raidgar98/skn_getin
# Kmochocki/music

## What has changed?

Replace random letters with music artists

## How it was achieved?

By finding nice artist and insetring them on list

## How to test it?

Verify checksums before and after edit

fixes https://github.com/raidgar98/skn_getin/issues/2
